packages/framework-integration-tests/package.json

```diff
@@ -45,6 +45,7 @@
   "main": "dist/index.js",
   "repository": "boostercloud/booster",
   "scripts": {
+    "watch:local": "nodemon --watch ../framework-provider-local/dist --watch ../framework-provider-local-infrastructure --watch dist --exec \"../cli/bin/run start -e local\"",
```

Comment:
This is useful for running the project locally with live reload

Reply:
🧙 🤘